src/strategies/citydao/examples.json

```diff
@@ -0,0 +1,19 @@
+[
+  {
+    "name": "Example query",
+    "strategy": {
+      "name": "citydao",
+      "params": {
+        "proofOfHumanityContract": "0xC5E9dDebb09Cd64DfaCab4011A0D5cEDaf7c9BDb",
+        "cityDaoCitizenNftContract": "0x40e27947a914e683089c6925313f782079b6f7f6",
```

Comment:
This isn't the CityDAO Citizen NFT it's just some random erc1155 that Vitalik Buterin happens to hold